fuzzing/common.bzl

```diff
@@ -50,6 +53,10 @@ Rule for creating a script to run the fuzzing test.
             executable = True,
             cfg = "host",
         ),
+        "_engine": attr.label(
```

Comment:
Maybe add a constraint that this label should offer the `BuildSettingInfo` provider?